compiler/luci-interpreter/src/kernels/Softmax.cpp

```diff
@@ -60,5 +75,17 @@ void Softmax::evalFloat() const
                                  getTensorShape(output()), getTensorData<float>(output()));
 }
 
+template <typename T> void Softmax::evalQuantized() const
+{
+  tflite::SoftmaxParams params{};
+  // params.beta = _params.beta;
+  params.table = const_cast<float *>(_table);
```

Comment:
Is `const_cast` necessary?

Reply:
I think so, cause if I run without `const_cast` it throw error like below
```
invalid conversion from ‘const float*’ to ‘float*’
```

Reply:
My mistake. We need `const_cast` because `execute` is a const member function. Please leave `const_cast`. Thanks.